api/src/libs/gh-utils/gh-requests.js

```diff
@@ -0,0 +1,99 @@
+//
+// Realm-o-Matic
+//
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Created by Shelly Xue Han
+//
+
+'use strict';
+
+import _ from 'lodash';
+import { GITHUB_REQUEST, GITHUB_JSON_PATH } from '../../constants';
+import shared from '../shared';
+import config from '../../config';
+import { jsonReader } from './gh-helpers';
+
+/**
+ * Create GitHub client with repo properties:
+ * @param {Function} ghFn Octokit function
+ * @param {Object} input extra properties
+ * @param {Object} outPaths paths to the respose object
+ */
+export const ghClient = async (ghFn, input, outPaths) => {
```

Comment:
This doesn't strike me as a `Client` so much as it is a Wrapper. For example, a client might have an API like `ghClient.fetchBranch(bla)` or `ghClient.createRef(bla)`.  In any case `input` are parameters for the ghFn so calling them `ghFnParams` or `params` would be more readable. I'd expect `outPath` to be some file or URL path but it looks like it can be an SHA. Can we find a more descriptive name of it?